### ISSUE TITLE
Feature request: support skip_plan in state migration

### ISSUE THREAD

**Issue by mdb** [minamijoyo/tfmigrate#151]
In addition to the `from_skip_plan` and `to_skip_plan` features supported by multi state migrations, it would be useful support a similar `skip_plan` option in single state migrations. I'm imagining something like:

```hcl
migration "state" "test" {
  dir = "dir1"
  skip_plan = true
  actions = [
    "mv aws_security_group.foo aws_security_group.foo2",
    "mv aws_security_group.bar aws_security_group.bar2",
  ]
}
```
## More context

Issue #142 outlines a use case for `from_skip_plan` and `to_skip_plan` in multi state migrations: 

> My team's `project-one` Terraform project is excessively monolithic and manages thousands of resources. As such, its `terraform plan` and `terraform apply` each take hours to complete. To fix this, I want to use `tfmigrate` to piecemeal and incrementally migrate `project-one` resources into more appropriately granular projects, like `project-two`. However, because `tfmigrate` performs a `terraform plan` against `project-one` with each `tfmigrate plan` and each `tfmigrate apply` -- and because `terraform plan`-ing `project-one` takes hours -- each `tfmigrate` action takes a prohibitively long time.
>
> In this case, I'd like to disable `tfmigrate` from `plan`-ing `project-one`. As such, I'm willing to accept the risks of bypassing the `terraform plan` diff analysis in favor of faster feedback and `tfmigrate` actions that don't take a prohibitively long time to complete.

I'd argue this use case is also applicable to single state migrations targeting very large, monolithic Terraform projects.

**Comment by mdb** [comment]
@minamijoyo Given that PR #152 addressed this issue, is it worth publishing a 0.3.17 `tfmigrate` release and closing this issue? Thanks!

**Comment by minamijoyo** [comment]
@mdb Thank you for your suggestion! I've cut a new release v0.3.17 🚀
